src/utils/formatting.ts

```diff
@@ -54,6 +54,17 @@ export const addQueryParams = (url: string, queryParams: { [key: string]: string
   return `${urlWithoutSearch}${queryString ? `?${queryString}` : ''}`;
 };
 
+export function removeQueryParam(key: string): string {
```

Comment:
Looks like we already have it in `#src/utils/location`

Reply:
that one uses location as a param, this one is intended to work with plain url

Reply:
Why not add a useLocation hook (assuming hook refactoring) and then you can use the existing function.

For the most part we are trying to prefer location, navigation, and searchParam hooks instead of direct `window.location.href` usage.

Reply:
yeah, initially i was going with location, but I guess because it's inside of a promise, the location state is not set as latest (prorably stores the location from the initial load when connecting to websocket notifications) and all the query params get deleted when you try to add or remove the `u` query param.

Reply:
Yes, makes sense, but I guess we should leave it for another cleanup update, because we have window.location in multiple places, and this bug fixing already end up with multiple unplanned changes.